### DISCUSSION TITLE
When I plug a second monitor into my notebook and I'm trying to draw with my Huion tablet, the pointer is in one place but xournall++ draw in other place

### DISCUSSION THREAD

**Discussion by geraldotnf** [xournalpp/xournalpp#3340] [Q&A]
When I plug a second monitor into my notebook and I'm trying to draw with my Huion tablet, the pointer is in one place but Xournal++ draw in other place a little bit to the left and to the bottom where I'm drawing. What I can do solve this?
OBS: When I used to use the old version of Xournal, I just had to to disable the option "use Xinput" to solve this problem, but I'm not founding this option in Xournal++.
I really apreciate it if someone could help me :)

**Comment by suisseWalter** [comment]
is your second monitor to the left/bottom of the normal one?
because with normal touch displays there is  a similar problem, where the touch input is streched over both monitors. this can be fixed by a xrandr command. but not sure if it's the same for tablets.

**Comment by Jus129** [comment]
I actually have the same problem where my second monitor when connected to my MSI laptop, the pen touch is actually a little offset(top right) around 4-5 cms from the cursor it's very frustrating to use, it's not in other drawing apps, I am not sure how to fix it. 0
is it because the aspect ratios of the screens are different? (The second monitor is square and I have a 15.6" laptop)

**Comment by Beliar25** [comment]
I have the same Problem with UgeeM708 on Windows indepentent of using second monitor or not. At the same time it is worthless working on linux. Got anyone found the way to fix it?

**Comment by Dartypier** [comment]
That happens because you have to set the area of interest of your tablet with the driver software. So install the Huion drivers for your tablet and in the software set the correct area of drawing.

**Reply by Beliar25** [reply]
Drivers made it actually mad. I installed it a long time ago, to set drawing area on main Display (Windows doesnt allow to draw on notebook display, not allowing to switch). There is actually no offset, if i turn xournal before the turning on the PenTablet app. I would use it without the app, cuz there is no offset , but in this case i cant use notebook display cuz of Windows. And if i start xournal with enabled pentablet process, then i have offset, and cant use interface correctly (after touching the tablet i cant use menus until i switch to other app and back to xournall) .

P.S. Using only 1 display there isnt any offset.

**Comment by ja-he** [comment]
In the case of X on Linux, e.g. `xinput` should be able to configure the device correctly.
You'll need your device ID for this afaik, which you can get from `xinput` (just search through the output, for my Huion pen display the name is `Tablet Monitor Pen Pen`).

Then you can map that to the correct output, again with `xinput`:
```sh
xinput map-to-output ${id} ${output}
# e.g. [...]           28   HDMI-A-0
```
